### ISSUE TITLE
[Request] wger fitness/workout 

### ISSUE THREAD

**Issue by deku-m** [alexbelgium/hassio-addons#61]
Hi,

I found an new docker/app might be interesting in your awesome add-on repo.
WGER is a Self hosted FLOSS fitness/workout, nutrition and weight tracker written with Django.
So for the sporty under us :) can create there own workout etc.

Repo:
https://github.com/wger-project/wger

**Comment by alexbelgium** [comment]
There is a potential image but it is amd64 only, so it won't be possible for me to test it. Are you open to help troubleshoot it?

**Comment by alexbelgium** [comment]
A first version is pushed! Ui should be available on port 8000. However the log would be interesting to already see if it starts ;)

**Comment by alexbelgium** [comment]
Hi, addon is up! UI lots and database is persistent. 

Could you please try some things to check if everything works? The key element is to restart the addon after you've done some elements as most issues are arising from lack of persistence after restart. Hopefully only the database needs to be persistent, otherwise just let me know!

Enjoy!

**Comment by alexbelgium** [comment]
Great news, thanks! I'll close the issue. Enjoy

**Comment by financelurker** [comment]
Hi! So, I'm late to the party... Has this AddOn been ditched from this repository? If so: why?

**Comment by alexbelgium** [comment]
Hi, it was indeed. The structure of the upstream image that prevents using root was incompatible to converting it as an addon. The previous version was functional but no data was remanent, which is not very useful

**Comment by deku-m** [comment]
Is it possible with the new version and docker images provided from wger to build a new one ?

**Comment by alexbelgium** [comment]
Nearly there on my test repo https://github.com/alexbelgium/hassio-addons-test but all static elements have 404... Can't understand why

**Comment by deku-m** [comment]
Looks like someone had the same:
https://github.com/wger-project/docker/issues/32

Can i test it as well?

**Comment by alexbelgium** [comment]
Of course, you can just install the repo above and search for wger... Thanks for the link I'll look at it

**Comment by deku-m** [comment]
I also notice the following:

/etc/cont-init.d/90-run.sh: executing
cp: cannot stat '/home/wger/src/static/*': No such file or directory
/etc/cont-init.d/90-run.sh: exiting 1

**Comment by alexbelgium** [comment]
Oops I'll remove that line sorry I was trying to aggregate all static folders I was seeing :)

**Comment by deku-m** [comment]
I update the image but still complaining about the it:

`/etc/cont-init.d/90-run.sh: executing
cp: cannot stat '/home/wger/src/static/*': No such file or directory
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
Done in 6.60s.
yarn run v1.22.19
$ sass wger/core/static/scss/main.scss:wger/core/static/yarn/bootstrap-compiled.css
Done in 2.17s.
Synchronizing exercises
System check identified some issues:
WARNINGS:
?: (axes.W001) You are using the django-axes cache handler for login attempt tracking. Your cache configuration is however invalid and will not work correctly with django-axes. This can leave security holes in your login systems as attempts are not tracked correctly. Reconfigure settings.AXES_CACHE and settings.CACHES per django-axes configuration documentation.
*** Synchronizing categories...
done!
*** Synchronizing muscles...`

**Comment by alexbelgium** [comment]
No worry about the complain, there is a "|| true" after it in the code so there is no impact

**Comment by deku-m** [comment]
I think it is something with the nGINX port as you changed the default web port as well.
In the topic he mentioned that that part also need to change.

**rolandgeider:**
If you want to change the port you need to change the nginx port in line 40 (the static files are served by it, otherwise you are accessing the application directly and all the css files are missing)

if you want to access the application over 8005, then simply to "8005:80"

https://github.com/wger-project/docker/blob/master/docker-compose.yml

**Comment by alexbelgium** [comment]
Hi, should be good now and I've pushed it to the main repo. Could you please test everything? Thanks

**Comment by alexbelgium** [comment]
Perfect, thanks for the feedback. It doesn't seem ingress can be implemented due to the app structure that uses some hardcoded url
